vllm/model_executor/layers/attention.py

```diff
@@ -85,7 +87,8 @@ def set_attn_bias(
             # Already set by a previous layer.
             return
         prompt_lens = input_metadata.prompt_lens
-        attn_bias = BlockDiagonalCausalMask.from_seqlens(prompt_lens)
+        attn_bias = BlockDiagonalMask.from_seqlens(
+            prompt_lens).make_local_attention(self.sliding_window)
```

Comment:
I have a quick question: Doesn't this change affect other models?

Reply:
aaah crap you're right. Some last minute changes ... Let me add a condition here !

Reply:
fixed I believe @WoosukKwon